Code/Framework/AzCore/AzCore/Math/VectorN.h

```diff
@@ -96,11 +96,11 @@ namespace AZ
         VectorN GetClamp(const VectorN& min, const VectorN& max) const;
         //! @}
 
-        //! Returns squared length of the vector.
-        float GetLengthSq() const;
+        //! Returns L1 norm (Manhattan distance) of the vector, full accuracy.
+        float L1Norm() const;
 
-        //! Returns length of the vector, full accuracy.
-        float GetLength() const;
+        //! Returns L2 norm (Euclidean distance) of the vector, full accuracy.
+        float L2Norm() const;
```

Comment:
I like the change, just mildly concerned it doesn't follow the convention

Reply:
Yeah, the worlds of numerical analysis and games are colliding a bit here.. Distinguishing between different types of norms is really important in many industries, including machine learning, but quite unfamiliar to game devs who are only used to dealing with L2 norms..